L1C_Aux/collocate_hirs_avhrr.py

```diff
@@ -69,13 +69,12 @@ def create_filepaths(args):
         fileobj = -1
 
         for date in [prev,in_date,next]:
-
             string_date = date[0:4]+'/'+date[4:6]+'/'+date[6:8]+'/'
 
-            for g in os.listdir(gac_dir+string_date):
-            
-                time_gac = rf.read_time(gac_dir+string_date+g)
 
+            
+            for g in os.listdir(gac_dir+string_date):
+                time_gac = rf.read_time(gac_dir+string_date+g)               
```

Comment:
On the diff all these bits come up as changes probably from where you have added/removed comments for debugging.  You can do a 'git diff' on your file before committing it to identify and remove these - it makes the reviewing easier.